include/CPULoadWidget.h

```diff
@@ -64,6 +65,9 @@ protected slots:
 
 	QTimer m_updateTimer;
 
+	int m_stepSize;
+	int stepSize() const { return m_stepSize > 0 ? m_stepSize : 1; }
```

Comment:
I'd go for `std::max` instead here. Also, I don't think we have a rule for it in our coding conventions, but we prefer to put all member functions before all member variables.